### DISCUSSION TITLE
Github handle for authors

### DISCUSSION THREAD

**Discussion by flaming-codes** [flaming-codes/crane-app#57] [Ideas]
# About

It would be nice to have the Github-handle for most authors, as most of the packages's code is also hosted on Github.

# Consideration

This information could be extracted from the author's repo, where the code might be hosted under the author's own account.

**Comment by lschoenm** [comment]
We could get the repo host from GitHub, but two problems come to my mind: 
- Packages often have multiple authors, in this case it wouldn't be possible to assign the host to the correct author
- Some packages are hosted by a company/research group/other handle 

Maybe we can find a solution to get some of the author's GitHub handles.
